src/Artsy/Relay/createRelaySSREnvironment.ts

```diff
@@ -40,6 +40,7 @@ interface Config {
 
 export interface RelaySSREnvironment extends Environment {
   relaySSRMiddleware: RelayClientSSR | RelayServerSSR
+  toggleFetching?: (isFetching) => void
```

Comment:
The parameter is missing its type annotation. I assume it’s a boolean?

```suggestion
  toggleFetching?: (isFetching: boolean) => void
```